services/workflows-service/prisma/schema.prisma

```diff
@@ -95,11 +95,18 @@ model EndUser {
   @@index([projectId])
 }
 
+enum BusinessPosition {
+  ubo
+  director
+  authorized_signatory
+}
+
 model EndUsersOnBusinesses {
   endUser    EndUser  @relation(fields: [endUserId], references: [id])
   endUserId  String
   business   Business @relation(fields: [businessId], references: [id])
   businessId String
+  positionInBusiness BusinessPosition[]
```

Comment:
Maybe `position` is a better name than `positionInBusiness`? It's already in the context of the relationship between an end user and a business, it makes sense to me. The same you call `user.email` and not `user.user_email`